### ISSUE TITLE
How do I get a Range slider for Sencha Touch 2?

### ISSUE THREAD

**Issue by kristianmandrup** [jepp/jep-Slider-for-Sencha-Touch-1.1#1]
Can I use this one or is there one included somehow? I tried passing an array of two values, but no luck.
Could you make a version of this jep-slider that works with ST2 ;) 
Thanks!

**Comment by jepp** [comment]
Yeah, I haven't bit that bullet yet.  I know they made a non-trivial amount of changes to the base slider component in ST2 (and the base components below that) which might make it a pain.  It was a pain to get all this working with ST1.

It's on my to-do but I'm not sure when I'll have time.

**Comment by kristianmandrup** [comment]
To me the Slider in ST2 looks very incomplete, with no tooltip (or other visualization) for the current value built in, no coloring of the bar for the current range selected etc. No support for range. I sure hope they upgrade the Slider in an upcoming release!
